src/Components/Bookpage/BookDetails/bookdetails.css

```diff
@@ -0,0 +1,15 @@
+.book-details {
+    display: flex;
+    max-width: 1200px;
+    margin: 3rem auto;
+    gap: 3rem;
+    flex-direction: row;
+    align-items: start;
```

Comment:
you should use flex-start instead of flex. Start works in some browsers, but not all. flex-start is widely supported